docs/tutorialkit.dev/src/components/Layout/Head.astro

```diff
@@ -16,8 +16,10 @@ import Default from '@astrojs/starlight/components/Head.astro';
     // @ts-ignore
     dataLayer.push(arguments);
   }
+
   // @ts-ignore
```

Comment:
@Nemikolh Would be cool if we can explain why we do this. Feel free to push to this branch.
```
// @ts-ignore because ...
```

Reply:
Comment added! :raised_hands:

Reply:
Thanks!

Reply:
For consistency we could use https://typescript-eslint.io/rules/ban-ts-comment/#allow-with-description

Reply:
Love that idea @AriPerkkio 🔥

Reply:
That is something we should even add to our eslint plugin IMHO.

Reply:
When exactly do these `@ts-ignore` here become required? If I remove them from this PR, all tests and builds pass just fine.

Reply:
Oh right, `pnpm docs:build` did fail https://github.com/stackblitz/tutorialkit/actions/runs/10004035387/job/27651984509?pr=152